ref/Microsoft.Build.Tasks.Core/net/Microsoft.Build.Tasks.Core.cs

```diff
@@ -134,6 +134,7 @@ public partial class CallTarget : Microsoft.Build.Tasks.TaskExtension
         [Microsoft.Build.Framework.OutputAttribute]
         public Microsoft.Build.Framework.ITaskItem[] TargetOutputs { get { throw null; } }
         public string[] Targets { get { throw null; } set { } }
+        [System.ObsoleteAttribute("Not supported")]
```

Comment:
I believe this counts as a breaking change since it adds a warning. Do we need it? We have a window for that but it's almost entirely closed now.

Reply:
The field is currently not implemented. I'd rather we never implement it to keep things simple. Instead of deprecating should I change the comments for these fields to something like `Not implemented, never will, forget you've seen this`